docs/gettingstarted/structuring.md

```diff
@@ -25,6 +25,69 @@ What is `@Mod`?
+
+Components:
+
+`modid` [Required]
+This identifies your mod uniquely to Forge and should probably be the first thing you enter. It's only visible internally.
+
+Example: @Mod(modid="examplemod")
+
+`name` [Required]
+This is the public name of your mod, as the user sees it.
+
+Example: @Mod(modid="examplemod", name="Mad Mod For Kool Kids")
+
+`version` [Required]
+This is the version of your mod and gets both publically displayed and internally used to detect version mismatches.
+You can write pretty much whatever in here, but it's generally a good idea to have some sort of structure. Even if it's something as simple as a single number that increases every time you update your mod.
+
+Example: @Mod(modid="examplemod", name="Mad Mod For Kool Kids" version="1.radical")
+
+`acceptableRemoteVersions` [Optional]
+This lets Forge know what Client-side versions your mod is ok with. The wildcard * (a star) means that your mod is fine with ANY version on the other end, even nothing at all. You can use this for Server-side only mods.
+[TODO: Someone add here how the versions can be structured. I THINK you can set minimum/maximum versions?]
+
+Example: [TODO]
+
+`canBeDeactivated` [Optional]
+This lets Forge know whether or not your mod can be safely turned off/on during gameplay. There's a menu option for enabling/disabling mods in the main menu, which uses this. Unlike most other annotation options this one takes a boolean instead of a string, so it's without quotes.
+Beware: This should NEVER be true for mods that add things to the game, like items or blocks. Things will go sideways if that happens. Some things cannot be taken back.
+
+Example: @Mod(canBeDeactivated=true)
+
+`clientSideOnly` [Optional]
+This prevents the mod from being loaded on the server at all, no matter what. If this is set together with the option below the universe will explode. Or the game crashes. One or the other.
+
+Example: @Mod(clientSideOnly=true)
+
+`serverSideOnly` [Optional]
+The opposite of the option above. This prevents the mod from being loaded on client-side. Beware: This also includes the internal server in singleplayer.
+
+Example: @Mod(serverSideOnly=true)
+
+`dependencies` [Optional]
+This defines what other mods yours depends on. Possibly also the reverse. Multiple entries can be made here, separated by a ; (What is this called?).
+[TODO: There seems to be some additional constructs and structures here that someone needs to expand on. Is there something other than "required-after"?]
+
+Example: @Mod(dependencies="required-after:Forge@[12.16.0.1859,);")
+
+`acceptedMinecraftVersions` [Optional]
+This tells Forge what versions of Minecraft your mod runs on. By default it's set to "whatever I'm currently building my mod against in my editor of choice". Tends to be quite specific. Sub-versions are not naturally included, so keep an eye out there.
```

Comment:
The default is actually set to "don't care, all versions". But ForgeGradle sets this property when building your mod.